src/main/java/uk/gov/hmcts/reform/wataskmonitor/services/CamundaService.java

```diff
@@ -1,28 +1,29 @@
-package uk.gov.hmcts.reform.wataskmonitor.services.jobs.configuration;
+package uk.gov.hmcts.reform.wataskmonitor.services;
 
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.reform.wataskmonitor.clients.CamundaClient;
-import uk.gov.hmcts.reform.wataskmonitor.models.camunda.CamundaTask;
-import uk.gov.hmcts.reform.wataskmonitor.services.utilities.ResourceUtility;
+import uk.gov.hmcts.reform.wataskmonitor.domain.camunda.CamundaTask;
+import uk.gov.hmcts.reform.wataskmonitor.utils.ResourceUtility;
 
 import java.time.ZonedDateTime;
 import java.time.format.DateTimeFormatter;
 import java.util.List;
 
-import static uk.gov.hmcts.reform.wataskmonitor.services.jobs.RequestParameterEnum.CONFIGURATION_JOB_SERVICE;
+import static uk.gov.hmcts.reform.wataskmonitor.services.jobs.RequestsEnum.CONFIGURATION_JOB_SERVICE;
+import static uk.gov.hmcts.reform.wataskmonitor.services.jobs.RequestsEnum.DELETE_PROCESS_INSTANCES_JOB_SERVICE;
 
 @Component
 @Slf4j
-public class ConfigurationJobService {
```

Comment:
This is a specific service for the configurationJob, the former name seems more appropriate

Reply:
I'd rather have a specific service per job rather than one big CamundaService for all of them.

Reply:
this is pending, Alex